### DISCUSSION TITLE
Any way to search/filter data?

### DISCUSSION THREAD

**Discussion by ifeldshteyn** [allinurl/goaccess#2612] [General]
Hi,

Is there any way to search/filter the data column? It gets quite tedious to switch pages (especially due to the 24 item limit). If not it would be a **really** useful feature. A simple use-case would be knowing what files people load when they access a folder.

**Comment by allinurl** [comment]
Check out #117. Currently working on it and making good progress! [**Stay tuned**](https://twitter.com/goaccess?lang=en)!
